lib/urbanopt/geojson/feature.rb

```diff
@@ -0,0 +1,94 @@
+module URBANopt
+  module GeoJSON
+    class Feature
```

Comment:
Cool, we might want to subclass Feature for each of our different Feature Types (Building, District System, etc)

Reply:
Would each feature type have to get treated differently? -- Not currently seeing any differences, but maybe I'm missing something? :)

Reply:
Yes I think you would subclass Feature for each of the different types

Reply:
We could try to share as much functionality as possible in the Feature base class.  But each sub class would have different properties and methods (because Buildings and DistrictSystems are different things)

Reply:
Does the creation of an abstract factory for the creation of these subclass objects sound like a step in the right direction to you? URBANopt::GeoJSON::GeoFile.new(path).get_feature(feature_id) would then return an instance of the Feature subclass rather than the Feature class.

Reply:
Sounds good :-)